### DISCUSSION TITLE
Unify the way we serve the website

### DISCUSSION THREAD

**Discussion by abdonrd** [Qiskit/qiskit.org#2370] [Ideas]
Right now, we have two kinds of deploys:

### Production

- Created with the `npm run generate`
- [Static deployment (pre-rendered)](https://nuxtjs.org/docs/2.x/get-started/commands#static-deployment-pre-rendered) on an IBM static server

### PR previews

- Created with the `npm run build`
- [Server deployment](https://nuxtjs.org/docs/2.x/get-started/commands#server-deployment) with `npm start` on Heroku

We have already had cases where one deploy worked and the other did not: https://github.com/Qiskit/qiskit.org/pull/2088#issuecomment-898732723

I want to propose to unify them: or both with static or both with server deployment.

Personally, since all of our content is static, I would lean towards the static deployment.

**Comment by korgan00** [comment]
This is not decoration. It is not related to aesthetics.

**Comment by Tansito** [comment]
Same opinion. Another issue related with this discussion was happening here #2399 , taking into account that static content offers really good performance compared to a server I agree going with the static option.

**Comment by vabarbosa** [comment]
updated preview build to use static deployment (similar to production), which also resolved the preview build errors:

- https://github.com/Qiskit/qiskit.org/pull/2401
